code/modules/awaymissions/zlevel.dm

```diff
@@ -58,7 +58,7 @@ var/global/list/potentialRandomZlevels = generateMapList(filename = "config/away
 			late_setup_level(block(locate(1, 1, zlev), locate(world.maxx, world.maxy, zlev)))
 			space_manager.remove_dirt(zlev)
```

Comment:
This *should* be handled by the `remove_dirt` hooks - there's one in particular that takes care of powernets. Is something amiss with that one?